app/models/event_user.rb

```diff
@@ -1,6 +1,6 @@
 class EventUser < ActiveRecord::Base
   # TODO Do we need these roles?
-  ROLES = [['Speaker', 'speaker'], ['Submitter', 'submitter'], ['Moderator', 'moderator']]
+  ROLES = [%w[Speaker speaker], %w[Submitter submitter], %w[Moderator moderator]]
```

Comment:
I followed the example of https://rubocop.readthedocs.io/en/latest/cops_style/#stylewordarray -I'm fixing it now